ctru-rs/src/test_runner.rs

```diff
@@ -0,0 +1,135 @@
+//! Custom test runner for building/running unit tests on the 3DS.
+
+extern crate test;
```

Comment:
Is this necessary?

Reply:
Yes, for example: https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=a37a055f3870e55bf55ba574a5ae6b86

At first I wasn't sure why, but I found this: https://doc.rust-lang.org/reference/names/preludes.html#extern-prelude 
The `test` crate is special since it is provided by rustc but not included in the extern prelude, so it must be declared with `extern crate` to be used, unlike typical Cargo crates.